config.go

```diff
@@ -39,47 +34,6 @@ type Config struct {
 	// Required. This, SigningKey or KeySetUrl(deprecated) or KeySetUrls.
 	SigningKeys map[string]interface{}
 
-	// URL where set of private keys could be downloaded.
```

Comment:
please update the README.md

Reply:
I've updated the `README.md`'s configuration table, import paths, and JWK Set test note.